extension/public/static/manifest/v3.json

```diff
@@ -3,6 +3,12 @@
   "version": "5.18.5",
   "version_name": "5.18.5",
   "description": "Freighter is a non-custodial wallet extension that enables you to sign Stellar transactions via your browser.",
+  "browser_specific_settings": {
+    "gecko": {
+      "id": "{3ee0dd4e-8c64-4b92-b539-25718a10f62f}",
+      "strict_min_version": "48.0"
```

Comment:
this is already present in the Manifest v2 json. We need to have this in v3 as well, specifically for Firefox